### PR TITLE
Azure 15.0.0-beta2 

### PR DESCRIPTION
**Pull request by whites11**
same as beta1 with downgraded flatcar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ to all Giant Swarm installations.
 
 - v15
   - v15.0
+    - [v15.0.0-beta2]((https://github.com/giantswarm/releases/tree/master/azure/v15.0.0-beta2)
```
Comment (fiunchinho) [inline README.md:131]: There is an extra `(` in both betas.

Comment (whites11) [reply README.md]: good catch

**Review by fiunchinho — APPROVED** [review]
should we archive `v15.0.0-beta1` instead of just deprecate?

**Review by T-Kukawka — APPROVED** [review]

**Comment by whites11** [comment]
> should we archive `v15.0.0-beta1` instead of just deprecate?

no, it's still in use.
After I release beta2, I'll upgrade `5jka7` and then archive

**Pull request merged**